pkg/types/types.go

```diff
@@ -106,9 +106,10 @@ type ResourceConfig struct {
 
 // DeviceSelectors contains common device selectors fields
 type DeviceSelectors struct {
-	Vendors []string `json:"vendors,omitempty"`
-	Devices []string `json:"devices,omitempty"`
-	Drivers []string `json:"drivers,omitempty"`
+	Vendors     []string `json:"vendors,omitempty"`
+	Devices     []string `json:"devices,omitempty"`
+	Drivers     []string `json:"drivers,omitempty"`
+	AcpiIndexes []string `json:"acpiIndexes,omitempty"`
```

Comment:
this probably needs to be under : GenericPciDeviceSelectors
we dont want it for Auxiliary device ATM (also i dont know if its even supported for it) hence lets limit it for now to PCI devices

Reply:
I just checked and to use scalable function inside a VM you can use VDPA so I think it's fine to move this to the PCI device section

https://metonymical.hatenablog.com/entry/2021/07/28/201833

Reply:
Moved the PCI device section